src/ui/camera.js

```diff
@@ -665,6 +665,7 @@ class Camera extends Evented {
      *     It does not correspond to a fixed physical distance, but varies by zoom level.
      * @param {number} [options.screenSpeed] The average speed of the animation measured in screenfuls
      *     per second, assuming a linear timing curve. If `options.speed` is specified, this option is ignored.
+     * @param {number} [options.maxDuration] The animation's maximum duration, measured in milliseconds.
```

Comment:
Let's clarify what happens if duration exceeds maxDuration in the docs.

Reply:
Thanks for the review! I've updated the PR.